### DISCUSSION TITLE
DNS troubles

### DISCUSSION THREAD

**Discussion by gcraenen** [techno-tim/k3s-ansible#182] [Questions] [answered]
After bringing the cluster up with these scripts and deploying some apps and pods in it, I am having problems with DNS in the cluster. I cannot reach pods, etc. in the cluster itself (same namespace) by dns name, but can reach them by ip address. Same goes for destinations outside of the cluster.

I tries and tested the steps from kubernete.io with the dnsutils pod and in those test I only find a lot of error messages like: 

[WARNING] No files matching import glob pattern: custom/*.override

It seems to be a coredns problem, already described in this blogpost: https://blog.differentpla.net/blog/2022/02/25/pod-dns-problems/

**Comment by timothystewart6** [comment, marked as answer]
Hey, this could be related to your DNS suffix in DHCP.  I would disabled it if you have it enabled for that VLAN and then test it again

**Reply by gcraenen** [reply]
Ok, I'm going to find out if I have that enabled in my local DNS server.

**Reply by gcraenen** [reply]
That was it!
